content/docs/tech-resources/deep-links-self-serve.md

```diff
@@ -1100,22 +1100,22 @@ curl https://your-tracking-domain-here/.well-known/assetlinks.json
 
 You can view your encoded links using Gmail, by selecting the three dots menu top-right, then "Show Original". Check the domain is as expected, and the `<A HREF ..` links are using HTTPS. Check that clicking the link takes you to the expected landing page.
 
-You can then use `curl` to request the tracked link, with the `-v` option to see the details of the tracked link request and response; see [here](./enabling-https-engagement-tracking-on-sparkpost#troubleshooting).
+You can then use `curl` to request the tracked link, with the `-v` option to see the details of the tracked link request and response; see [here](./enabling-https-engagement-tracking-on-sparkpost#troubleshooting-tips).
 
 ### Check your app matches your tracking domain
 
 The domains entitlement in your app(s) must match your tracking domain. This can be done specifically, or (with Apple) a wildcard matching sub-domain. Refer to
-* [Apple](#ios-spec-file) configuration
+* [Apple](#ios-example-apple-app-site-association) configuration
 * [Android](android-spec-file) configuration
 
 
-### <a name="testing-auto-verify"></a>Android: testing auto-verify
+### Android: testing auto-verify
 
 Getting your app to auto-verify requires the `assetlinks.json` file to be available on your _specific_ tracking domain rather than relying on the organizational domain (main website).
 
-To test auto-verify works for your domain(s), start with a fresh install of your app, as mentioned in [this article](#auto-verify-article), because Android remembers the user's choice.
+To test auto-verify works for your domain(s), start with a fresh install of your app, as mentioned in [this article](#article-on-android-deep-links-https-levelup-gitconnected-com-the-wrong-hacked-and-correct-way-of-android-deep-linking-for-redirected-multisite-with-autoverify-5c72fb1f8053-the-wrong-and-right-way-specifically-auto-verify-on-variations-of-your-different-domains), because Android remembers the user's choice.
```

Comment:
So, I tried clicking on this link, and it doesn't work. The reason here is because the "deep links" inside of a page work with heading type tags (so `#` or `##` for example). The link trying to be referenced in this case is going to an ordered list, which won't work with a deep link.

However, since this deep link is trying to go to the following text "Article on [Android deep links] the wrong and right way (specifically, auto-verify on variations of your different domains)", wouldn't it be better to just send them directly to the external link? (https://levelup.gitconnected.com/the-wrong-hacked-and-correct-way-of-android-deep-linking-for-redirected-multisite-with-autoverify-5c72fb1f8053)